Cargo.toml

```diff
@@ -20,7 +20,7 @@ include = [
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "0.19"
+tree-sitter = "~0.20"
```

Comment:
tree-sitter-cli version is still at 0.19.5 in https://github.com/tree-sitter/tree-sitter-scala/blob/0ceab66a43914320da893199ecd760473c043769/package.json#L16

Should these generally match up?

Reply:
I'm not very familiar with tree-sitter-cli unfortunately... but if there is a later version, then probably worth updating to it anyways

Reply:
Yea. I am also not sure about the implication of updating either of these versions, so could we remove this change out of the PR, since the main point is Rust code changes?

Reply:
> Yea. I am also not sure about the implication of updating either of these versions, so could we remove this change out of the PR, since the main point is Rust code changes?

updating the version of tree sitter in the cargo.toml was actually the main point for me. It was blocking scala integration for lapce. This is important for the rust binding.

Reply:
The newest one is `0.20.9`, btw.
https://crates.io/crates/tree-sitter

Reply:
ok. Then let's bump both up and see if the tests pass etc?

Reply:
> ok. Then let's bump both up and see if the tests pass etc?

Actually... since I've deleted my repository, this is kinda weird. Clearly github still has all the code and stuff but I don't actually have any way to make commits. I think maintainers might be able to make commits though? Unsure

Reply:
Note that the crates version doesn't actually match the latest cli version, which is `0.20.7`.

@VixieTSQ since you created the PR on your master branch the maintainers here actually can't add commits to it. It would need to be another branch. Do you mind actually creating a new PR with these changes that would supercede this one?

Reply:
Oh yes,
https://crates.io/crates/tree-sitter-cli -> `0.20.7`
Thanks